AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -330,17 +350,26 @@ private extension UPIComponent {
             continueButton.title = localizedString(.QRCodeGenerateQRCode, configuration.localizationParameters)
         }
         
-        updateButtonState()
-    }
-    
-    func updateButtonState() {
-        continueButton.enabled = canSubmit()
+        hideError()
     }
     
     func focusVpaInput() {
         vpaInputItem.focus()
     }
     
+    private func showError() {
+        errorItem.isHidden.wrappedValue = false
+        UIAccessibility.post(
+            notification: .announcement,
+            argument: "\(localizedString(.errorTitle, configuration.localizationParameters)): \(errorItem.message ?? "")"
+        )
+    }
+
+    private func hideError() {
+        errorItem.message = nil
```

Comment:
If set to nill and need to re-show, message will be gone

Reply:
Ohh good catch! will adjust the tests to catch it